metadata.json

```diff
@@ -29,7 +33,7 @@
       "operatingsystem": "OracleLinux",
       "operatingsystemrelease": ["5","6","7"]
     },
-   {
+    {
```

Comment:
this looks like a typo

Reply:
Re-checked. This is correcting a real alignment issue; the misalignment occurred in ba150a5.